infoportal-client/src/shared/AgeGroupTable.tsx

```diff
@@ -17,12 +17,15 @@ export const AgeGroupTable = ({
   persons,
   hideHeader,
   enableDisplacementStatusFilter,
+  enableOnlyPwdFilter,
   ...sx
 }: {
   hideHeader?: boolean
   enableDisplacementStatusFilter?: boolean
   tableId: string
   persons?: PersonDetails[]
+  enableOnlyPwdFilter?: boolean;
```

Comment:
The name is confusing. "only" seems to indicate that it will override `enableDisplacementStatusFilter`. I think `enablePwdFilter` is more suited.